test/unit/spec/nbextensions/bulkImportCell/tabs/fileTypePanel-spec.js

```diff
@@ -161,5 +161,35 @@ define([
                     // so it doesn't dangle.
                 });
         });
+
+        [true, false].forEach((viewOnly) => {
+            it(`should${viewOnly ? ' not' : ''} show icons when${
+                viewOnly ? '' : ' not'
+            } in view only mode`, async () => {
+                const panel = FileTypePanel.make({
+                    bus: Runtime.make().bus(),
+                    fileTypes,
+                    header,
+                    toggleAction: () => {},
+                    viewOnly,
+                });
+                await panel.start({
+                    node: container,
+                    state: {},
```

Comment:
Default state is everything is incomplete, so there should be `fa-exclamation` icons.